src/components/component-playground.jsx

```diff
@@ -550,38 +553,46 @@ module.exports = React.createClass({
     });
   },
 
-  _getFilteredComponents() {
-    return _.mapValues(this.props.components, function(fixtures) {
-      return _.assign({}, fixtures, {
+  _getComponentsWithMatchingFixtures() {
+    return _.pick(_.mapValues(this.props.components, function(details) {
+      return _.assign({}, details, {
       // Only select fixtures matching search text and rebuild the object.
-        fixtures: _.pick(fixtures.fixtures,
+        fixtures: _.pick(details.fixtures,
```

Comment:
`filter` returns an array containing the filtered items' values.
`pick` returns an object containing the filtered key: value pairs.

So, IMO, `pick` seems better here to preserve the initial structure.